hack/machineset.sh

```diff
@@ -79,9 +79,9 @@ get_aws_ms() {
   local provider=$4
 
   # get the AMI id for the Windows VM
```

Comment:
Consider updating these [docs](https://github.com/openshift/windows-machine-config-operator/blob/256c3e5159bafb380db364f386b04fbc9e4fed9f/docs/machineset-aws.md#creating-an-aws-windows-machineset) too.

Reply:
yes still in the works.